app/containers/AccountInitializer/tests/reducer.test.js

```diff
@@ -0,0 +1,8 @@
+import accountInitializerReducer from '../reducer';
```

Comment:
Missing tests for other reducer actions. Example:
https://github.com/react-boilerplate/react-boilerplate/blob/master/app/containers/HomePage/tests/reducer.test.js